src/ring_theory/power_series/well_known.lean

```diff
@@ -83,4 +84,72 @@ variables {A A'} (n : ℕ) (f : A →+* A')
 
 end field
 
+open ring_hom
+open finset nat
+
+variables (A : Type*) [comm_ring A]
+
+noncomputable def eval_mul_hom (a : A) : power_series A →+* power_series A :=
+{ to_fun :=   λ f, mk $ λ n, a^n * (coeff A n f),
+  map_zero' := by { ext, simp only [linear_map.map_zero, coeff_mk, mul_zero], },
+  map_one' := by { ext1, simp only [mul_boole, coeff_mk, coeff_one], split_ifs,
+                rw [h, pow_zero], refl, },
+  map_add' := by {intros, ext, norm_num, rw mul_add, },
+  map_mul' := by {intros, ext, rw [coeff_mul, coeff_mk, coeff_mul, finset.mul_sum],
+                apply sum_congr rfl, norm_num, intros b c H, rw [<-H, pow_add], ring, }, }
+
+/-- Shows that `e^(aX) * e^(bX) = e^((a + b)X) ` -/
+theorem exp_mul_exp_eq_exp_add [algebra ℚ A] (a b : A) :
+  eval_mul_hom A a (exp A) * eval_mul_hom A b (exp A) = eval_mul_hom A (a + b) (exp A) :=
```

Comment:
eg with the change above, this becomes
```suggestion
  eval_mul_hom a (exp A) * eval_mul_hom b (exp A) = eval_mul_hom (a + b) (exp A) :=
```

Reply:
I see. I have made the change, thank you!